ui/src/Pages/Preferences/Invites/ManageInvites.jsx

```diff
@@ -10,6 +10,8 @@ import {
 import TrashIcon from "../../../assets/Icons/Trash";
 
 function ManageInvites() {
+  const href = window.location.host;
```

Comment:
This constant can be defined in the toClipboard callback as its only used there.